components/list/src/string_matcher.rs

```diff
@@ -138,12 +138,13 @@ impl<'data> StringMatcher<'data> {
         })
     }
 
+    #[allow(clippy::unwrap_used)] // by invariant
```

Comment:
this was hitting the attribute macro error on newer nightlies, probably due to changes in the matches expansion